Bootstrap/Update.php

```diff
@@ -223,4 +224,18 @@ private function clearConfigCache()
             $cacheManager->clearConfigCache();
         }
     }
+
+    private function updateConnectAttribute(){
+        if (version_compare($this->version, '1.0.6', '<=')) {
```

Comment:
I also changed plugin version to 1.0.6, but my branch is not merged yet. So your has higher priority :)